pom.xml

```diff
@@ -120,6 +115,27 @@
           <version>1.3</version>
           <scope>test</scope>
         </dependency>
+        <dependency> 
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>git</artifactId>
+          <version>2.6.5</version>
+          <classifier>tests</classifier>
+          <scope>test</scope>
+        </dependency>
+        <dependency>
+          <groupId>org.jenkins-ci.plugins.workflow</groupId>
+          <artifactId>workflow-multibranch</artifactId>
+          <version>2.11</version>
```

Comment:
`2.11` requires `step-api` at `2.7`, so I think it should be explicitly added or just update `workflow-aggregaror` to `2.5`.